CMakeLists.txt

```diff
@@ -127,6 +127,7 @@ find_package (PythonInterp REQUIRED)
 find_package(Gettext REQUIRED)
 find_package(OpenGL REQUIRED)
 find_package(PNG REQUIRED)
+find_package(CURL REQUIRED)
```

Comment:
Take a look at lines 91 to 113. We need to tell mingw explicitly to use the static libraries. Otherwise libcurl.dll.a is used.
Try adding
```
set(CURL_LIBRARY "C:/msys64/mingw32/lib/libcurl.a" CACHE INTERNAL "")
```